site/index.html

```diff
@@ -37,31 +37,16 @@ <h1>
 
   <div class="promo_table">
       <div class="promo_table_cell">
-          <h3>Autosuggestions</h3>
-
-          <div class="promo_image">
-              <a target="_blank" href="assets/img/screenshots/autosuggestion.png">
-                  <img alt="fish shows suggestions based on files, completions and your history. In this case it offers to turn 'ssh -' into 'ssh -l demo SomeLongHost@SomeLongDomainIAlwaysMisspell.com', and you can accept it with right arrow or control-F" src="assets/img/screenshots/autosuggestion.png">
-              </a>
-          </div>
-
-          fish suggests commands as you type based on history and completions, just like a web browser.
-          Watch out, Netscape Navigator 4.0!
-      </div>
-
-      <div class="promo_table_cell">
-          <h3>Glorious VGA Color</h3>
+          <h3>Works Out of the Box</h3>
 
           <div class="promo_image">
-              <a target="_blank" href="assets/img/screenshots/colors.png">
-                  <img alt="A colorful commandline, with syntax highlighting" src="assets/img/screenshots/colors.png">
+              <a target="_blank" href="assets/img/screenshots/works_out_of_the_box.png">
+                  <img alt="A git clone followed by the completions for 'git checkout --track origin/', showing the origin's remote branches" src="assets/img/screenshots/works_out_of_the_box.png">
               </a>
           </div>
 
-          fish supports 24 bit true color, the state of the art in terminal technology.
-		  Behold the monospaced rainbow.
+          fish will delight you with features like tab completions and syntax highlighting that just work. It will make you exclaim "Of course fish does that!".
```

Comment:
I like the old wording of "nothing new to learn or configure." but I guess this is unequivocal too